src/tlo/methods/tb.py

```diff
@@ -1721,6 +1721,7 @@ def __init__(self, module, person_id, suppress_footprint=False):
         self.TREATMENT_ID = "Tb_Test_Screening"
         self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"Over5OPD": 1})
         self.ACCEPTED_FACILITY_LEVEL = '1a'
+        self.EQUIPMENT = set()  # no specific equipment required
```

Comment:
```suggestion
        self.EQUIPMENT = set()  # no specific equipment required unless updated later
```

Reply:
We may just comment here that it might be changed later for this HSI event. What do you think?

Reply:
If you agree, the same should be done for HSI_Tb_FollowUp as well.